packages/shared/src/components/fields/Dropdown.tsx

```diff
@@ -124,7 +124,9 @@ export function Dropdown({
       <Menu
         disableBoundariesCheck
         id={id}
-        className={`menu-primary ${scrollable && 'scrollable'}`}
+        className={`${
+          scrollable ? 'menu-secondary scrollable' : 'menu-primary'
```

Comment:
Why do we set the menu to secondary if it's scrollable? Shouldn't we support scrollable in both modes?

Reply:
Secondary is basically the menu that has the white children rendered.
The scrollable menu could technically be both yes, but for now this is the only one that we have.

And for this fix we need it to be on the secondary one.
I think we will revisit this in the rework on the form layout fields as they will change back anyway.

Reply:
It's easy to apply this support for both cases isn't it? If so it's better to do it now. This is not trivial to create this edge case. It can cause confusion in the future

Reply:
Ok in that case I have to introduce which type of menu it needs to render.
As now the dropdown is always primary.

Will rework this, agreed it might cause confusion